src/sentry/api/endpoints/release_thresholds/release_threshold_status_index.py

```diff
@@ -126,61 +161,161 @@ def get(self, request: Request, organization: Organization | RpcOrganization) ->
+
+                TODO: determine whether results have been truncated or not
+                - this will determine whether we can give an accurate status report or not
+                """
+                error_counts = get_errors_counts_timeseries_by_project_and_release(
+                    end=end,
+                    environments_list=environments_list,
+                    organization_id=organization.id,
+                    project_id_list=project_id_list,
+                    release_value_list=release_value_list,
+                    start=start,
+                )
+                for ethreshold in category_thresholds:
+                    # TODO: filter by environment as well?
+                    is_healthy = is_error_count_healthy(ethreshold, error_counts)
+                    ethreshold["is_healthy"] = is_healthy
+                    release_threshold_health[ethreshold["key"]].append(
+                        ethreshold
+                    )  # so we can fill all thresholds under the same key
+            elif threshold_type == ReleaseThresholdType.NEW_ISSUE_COUNT:
+                for ethreshold in category_thresholds:
+                    release_threshold_health[ethreshold["key"]].append(
+                        ethreshold
+                    )  # so we can fill all thresholds under the same key
+            elif threshold_type == ReleaseThresholdType.UNHANDLED_ISSUE_COUNT:
+                for ethreshold in category_thresholds:
+                    release_threshold_health[ethreshold["key"]].append(
+                        ethreshold
+                    )  # so we can fill all thresholds under the same key
+            elif threshold_type == ReleaseThresholdType.REGRESSED_ISSUE_COUNT:
+                for ethreshold in category_thresholds:
+                    release_threshold_health[ethreshold["key"]].append(
+                        ethreshold
+                    )  # so we can fill all thresholds under the same key
+            elif threshold_type == ReleaseThresholdType.FAILURE_RATE:
+                for ethreshold in category_thresholds:
+                    release_threshold_health[ethreshold["key"]].append(
+                        ethreshold
+                    )  # so we can fill all thresholds under the same key
+            elif threshold_type == ReleaseThresholdType.CRASH_FREE_SESSION_RATE:
+                for ethreshold in category_thresholds:
+                    release_threshold_health[ethreshold["key"]].append(
+                        ethreshold
+                    )  # so we can fill all thresholds under the same key
+            elif threshold_type == ReleaseThresholdType.CRASH_FREE_USER_RATE:
+                for ethreshold in category_thresholds:
+                    release_threshold_health[ethreshold["key"]].append(
+                        ethreshold
+                    )  # so we can fill all thresholds under the same key
```

Comment:
these all have the same logic. should the appending be moved outside of all the if statements?

Reply:
Ah yea - for now this is just a placeholder until the other threshold types are implemented.
Since this will continue being a WIP/feature flagged i'm not too hardpressed to unify this (eventually they'll all likely be pretty different)